sandboxes/dynamicanalysis/Dockerfile

```diff
@@ -0,0 +1,129 @@
+# NO_PUBKEY EB3E94ADBE1229CF [powershell]
+# NO_PUBKEY B53DC80D13EDEF05 [kubectl]
+
+# buildpack-deps 22.04 [ubuntu]
+FROM buildpack-deps@sha256:d76488a56e1e85930e053b794daa9f990636565bf0de54903d65d93c43c3cad5 AS image
+
+ENV DEBIAN_FRONTEND noninteractive
+# Silence 'debconf: delaying package configuation, since apt-utils is not installed' warning
+# as it related to interactive configuration anyway
+ENV DEBCONF_NOWARNINGS="yes"
+
+# Essentials
+RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-recommends \
+	apt-transport-https \
+	npm \
+	php \
+	php-zip \
+	php-gd \
+	python3-pip \
+	ruby \
+	ruby-rubygems \
+	rust-all \
+	software-properties-common
+
+# set repo for kubectl
+RUN mkdir -m 0700 -p /etc/apt/keyrings && \
+ 	curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg -o /etc/apt/keyrings/kubernetes.gpg && \
+ 	echo "deb [signed-by=/etc/apt/keyrings/kubernetes.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list
+
+# install keys for powershell
+RUN curl -fsSL "https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb" -o ./packages-microsoft-prod.deb && \
+	dpkg -i packages-microsoft-prod.deb
+
+# Extra packages for realistic runtime
+RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-recommends \
+	auditd \
+	awscli \
+	build-essential \
+	cmake \
+	clang \
+	golang \
+	iproute2 \
+	iputils-ping \
+	kubectl \
+	libpng-dev \
+	libzip-dev \
+	net-tools \
+	netcat \
+	powershell \
+	protobuf-compiler \
+	python2 \
+	sshpass \
+	sudo \
+	tcpdump \
+	telnet \
+	tshark \
+	zip
+
+# Configure sudo for passwordless execution
+RUN echo "ALL ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+#
+# PHP
+# Install Composer
+RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
+    php -r "if (hash_file('sha384', 'composer-setup.php') === '55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
+    php composer-setup.php && \
+    php -r "unlink('composer-setup.php');" && \
+    mv composer.phar /usr/local/bin/
+
+COPY analyze-php.php /usr/local/bin/
+RUN chmod 755 /usr/local/bin/analyze-php.php
+
+#
+# NPM
+COPY analyze-node.js /usr/local/bin/
+RUN chmod 755 /usr/local/bin/analyze-node.js
+COPY bowerrc /app/.bowerrc
+
+#
+# Python
+# Some Python packages expect certain dependencies to already be installed
+COPY pypi-packages.txt ./
+RUN pip install -r pypi-packages.txt
+
+COPY analyze-python.py /usr/local/bin/
+RUN chmod 755 /usr/local/bin/analyze-python.py
+
+#
+# Rubygems
+COPY analyze-ruby.rb /usr/local/bin/
+RUN chmod 755 /usr/local/bin/analyze-ruby.rb
+
+#
+# Rust
+COPY analyze-rust.py /usr/local/bin/
+RUN chmod 755 /usr/local/bin/analyze-rust.py
+
+
+WORKDIR /app
+
+#
+# Rust (in /app)
+RUN cargo init
```

Comment:
CAn we keep all the language bits together? I.e. move this to be next to lines 99/100.

Reply:
Yes sure. I only separated it like that because of the `WORKDIR` command but I'll figure out what it should be reset to after